capa/v25.1.1/README.md

```diff
@@ -0,0 +1,20 @@
+# :zap: Giant Swarm Release v25.1.1 for CAPA :zap:
+
```

Comment:
`This release introduces several changes that are required for Vintage to CAPA migration use-cases.

Most notable change is that now auditd kernel setting is disabled by default. If you actively use this feature, please add following field global.components.auditd.enabled set to true in the [cluster values file](https://github.com/giantswarm/cluster/blob/main/helm/cluster/values.yaml#L31) before the upgrade.`